x-pack/plugins/security_solution/server/endpoint/services/actions/create/update_cases.test.ts

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { updateCases } from './update_cases';
+import type { CasesClient } from '@kbn/cases-plugin/server';
+import type { CreateActionPayload } from './types';
+
+jest.mock('@kbn/cases-plugin/server');
+const mockCasesClient = {
+  cases: {
+    getCasesByAlertID: jest.fn(),
+  },
+  attachments: {
+    bulkCreate: jest.fn(),
+  },
+} as unknown as CasesClient;
+
+describe('updateCases', () => {
```

Comment:
yeah, probably don't need this since it is going to be deleted in favor of `BaseResponseActionsClient.updateCase()` method - which already has tests: https://github.com/elastic/kibana/blob/5fa14686e5c6b3edffb338c3029a46a4316fdfca/x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.test.ts#L92

Reply:
That's good news ;) I love to remove the code (even thought I had to write it first) ;P